package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "hexo": {
-    "version": "3.7.0"
+    "version": "3.7.1"
```

Comment:
Usually we would want to keep the separate so it's topic-specific but that is the most recent version of hexo. 🤷🏼‍♂️

Reply:
Ah...I didn't remember doing anything to it...let me change it back real quick. Thanks!